test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -86,6 +86,54 @@ def validate_error(self, err):
                 "ERROR in submitting a job with future time: %s" %
                 err.msg[0])
 
+    def jobdir_shared_body(self, location):
+        """
+        Test submission of job with sandbox=PRIVATE,
+        and moms have $jobdir_root set to shared.
+        """
+        self.remove_jobdir = True
+        momA = self.moms.values()[0]
+        momB = self.moms.values()[1]
+
+        loglevel = {'$logevent': 4095}
+        momB.add_config(loglevel)
+        c = {'$jobdir_root': '%s shared' % location}
+        for mom in [momA, momB]:
+            mom.add_config(c)
+            mom.restart()
+
+        a = {'Resource_List.select': '2:ncpus=1',
+             'Resource_List.place': 'scatter',
+             ATTR_sandbox: 'PRIVATE',
+             }
+        j = Job(TEST_USER, attrs=a)
+        j.set_sleep_time(10)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+        attribs = self.server.status(JOB, id=jid)
+        jobdir = attribs[0]['jobdir']
+        self.jobdir.append(jobdir)
+        relpath = os.path.join(
+            self.server.pbs_conf['PBS_EXEC'], 'bin', 'pbs_release_nodes')
+
+        rel_cmd = [relpath, '-j', jid, momB.shortname]
```

Comment:
Shouldn't there be a PTL wrapper for pbs_release_nodes?

Reply:
@bhroam : no such wrapper currently exists for pbs_release_nodes.

Reply:
There really should be one.  Maybe you can file a ticket to get one written?

Reply:
@bhroam : ok, I'll file a ticket to get a PTL wrapper written for pbs_release_nodes.